### PR TITLE
Tony/seo create token adjustment

### PR DESCRIPTION
**Pull request by tonyboylehub**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,18 +1,26 @@
 {
     "cSpell.words": [
+        "Arweave",
         "blockhash",
         "callout",
+        "Devnet",
         "hashlist",
         "Hookable",
+        "irsy",
```
Comment (MarkSackerberg) [inline .vscode/settings.json:9]: This send to be wrong

**Review by MarkSackerberg — DISMISSED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (tonyboylehub) [reply .vscode/settings.json:9]: this is just the spell checker config. I'll sort that.

Comment (MarkSackerberg) [reply .vscode/settings.json:9]: That’s why I approved nevertheless

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,18 +1,25 @@
...
+        "Arweave",
         "blockhash",
         "callout",
+        "Devnet",
         "hashlist",
         "Hookable",
+        "Irys",
         "Keypair",
         "metaplex",
+        "metedata",
```
Comment (MarkSackerberg) [inline .vscode/settings.json:12]:
```suggestion
        "metadata",
```

**Review by MarkSackerberg — APPROVED** [review]

**Review by MarkSackerberg — APPROVED** [review]

**Pull request merged**